src/commands/flex/plugins/build.js

```diff
@@ -18,7 +17,7 @@ class FlexPluginsBuild extends FlexPlugin {
 }
 
 FlexPluginsBuild.description = createDescription(
-  'Builds your Flex plugin and creates a JavaScript and sourcemap bundle',
+  'Builds Flex plugin and creates a JavaScript and sourcemap bundle',
```

Comment:
should this be `Build the Flex plugin`? some of the them are in the format `{action} Flex plugin` while others are `{action} the Flex plugin`